### PR TITLE
Update keyboard-shortcuts.md

### PR DESCRIPTION
**Pull request by efb4f5ff-1298-471a-8973-3d47447115dc**
added shortcut form https://github.com/FreeTubeApp/FreeTube/pull/2427

### CODE REVIEW

File: usage/keyboard-shortcuts.md [hunk 1]
```diff
@@ -7,30 +7,31 @@ permalink: /usage/keyboard-shortcuts/
...
+| Play or Pause                                   | Play / Pause video                                                                                                                                                                                   |
+| Previous or Next                                | Skips to the Previous / Next video in a playlist                                                                                                                                                     |
+| , or .                                          | While the video is paused, skip to the previous / next frame.                                                                                                                                        |
+| SHIFT + Enter or SHIFT + Click on Search button | After user puts in search query use on of the following shortcuts to perform the search in a new instance                                                                                            |
```
Comment (ChunkyProgrammer) [inline usage/keyboard-shortcuts.md:34]: Could be a bit more concise
```suggestion
| SHIFT + Enter or SHIFT + Click on Search button | Search in a new instance of FreeTube                                                                                                                                         |
```

Comment (ChunkyProgrammer) [reply usage/keyboard-shortcuts.md:34]: Let me know your thoughts on this change

Comment (efb4f5ff-1298-471a-8973-3d47447115dc) [reply usage/keyboard-shortcuts.md:34]: Yes i completely agree. I didnt know how to properly word it.

File: usage/keyboard-shortcuts.md [hunk 2]
```diff
@@ -7,30 +7,31 @@ permalink: /usage/keyboard-shortcuts/
...
+| Play or Pause                                   | Play / Pause video                                                                                                                                                                                   |
+| Previous or Next                                | Skips to the Previous / Next video in a playlist                                                                                                                                                     |
+| , or .                                          | While the video is paused, skip to the previous / next frame.                                                                                                                                        |
+| SHIFT + Enter or SHIFT + Click on Search button | Search in a new instance of FreeTube                                                                                                                                         |
```
Comment (PikachuEXE) [inline usage/keyboard-shortcuts.md:34]: Search/Browse Video, Playlist etc. in a new window

**Review by PikachuEXE — CHANGES_REQUESTED** [review]
"new instance of FreeTube" sound wrong

File: usage/keyboard-shortcuts.md (hunk 2 above)
Comment (efb4f5ff-1298-471a-8973-3d47447115dc) [reply usage/keyboard-shortcuts.md]: https://github.com/FreeTubeApp/FreeTube-Docs/pull/33/commits/59eea202e3039fe0db180353a16982c648c14ab7

**Review by PikachuEXE — APPROVED** [review]

**Review by ChunkyProgrammer — APPROVED** [review]

**Review by absidue — APPROVED** [review]

**Pull request merged**